### PR TITLE
* Add test for fileupload to DM

### PR DESCRIPTION
**Pull request by vianvi**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/tests/legal/certificateOfService/uploadFile_test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Feature('verify certification Of service File features')
+
+Scenario('I can select options  and upload file @legal', function * (I, legalUserSteps, legalCertificateOfServiceSteps) {
+  const userEmail = yield I.createSolicitorUser()
+  legalUserSteps.loginAndStartClaim(userEmail)
+  legalCertificateOfServiceSteps.uploadDocument()
+  const downLoadUrl = yield I.grabAttributeFrom('ul.list-number.list-border li a', 'href')
+  console.log(downLoadUrl)
```
Comment (yarpo) [inline src/tests/legal/certificateOfService/uploadFile_test.js]: do we want to keep `console.log`?

File: src/tests/legal/certificateOfService/pages/what-documents.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+'use strict'
```
Comment (yarpo) [inline src/tests/legal/certificateOfService/pages/what-documents.js]: are you sure you don't want to add this file as a TypeScript?

Comment (vianvi) [reply src/tests/legal/certificateOfService/pages/what-documents.js]: not at this moment- all team 2 scripts are still in JS.Lot of refactor required, will do it in future.

File: src/tests/legal/certificateOfService/uploadFile_test.js (hunk 1 above)
Comment (vianvi) [reply src/tests/legal/certificateOfService/uploadFile_test.js]: yep -it helps us to understand it is actually going to DM for time being, will remove once we refactor our URL.

**Pull request merged**